_data/mentors.yml

```diff
@@ -194,4 +194,13 @@
   github: hossainalikram
   image: hossain.jpg
   twitter: Hossainalikram
+  bio: 
+  
+  - name: Doris Lee
```

Comment:
@roonyh @jig08 just checked the website. One of the mentors hossain's record has vanished. We need to fix that.